inc/views/base/calendar/addevent.twig

```diff
@@ -0,0 +1,404 @@
+{% extends 'layouts/master.twig' %}
+
+{% block head %}
+    <title>{{ mybb.settings.bbname }} - {{ lang.calendar }} - {{ lang.add_event }}</title>
+{% endblock head %}
+
+{% block body %}
+    {{ event_errors|raw }}
+    <form action="calendar.php" method="post" name="input">
+        <input type="hidden" name="my_post_key" value="{{ mybb.post_code }}" />
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder clear">
+            <tr>
+                <td colspan="2" width="100%" class="thead"><strong>{{ lang.add_event }}</strong></td>
+            </tr>
+            <tr>
+                <td width="20%" class="trow1"><strong>{{ lang.event_name }}</strong></td>
+                <td class="trow1"><input type="text" class="textbox" size="50" name="name" maxlength="120" value="{{ select.name }}"/></td>
+            </tr>
+            {% if calendarcount > 1 %}
+                <tr>
+                    <td width="20%" class="trow1"><strong>{{ lang.select_calendar }}</strong></td>
+                    <td class="trow1">
+                        <select name="calendar">
+                            {% for calendar in calendar_select %}
+                                <option value="{{ calendar.cid }}"{{ calendar.selected }}>{{ calendar.name }}</option>
+                            {% endfor %}
+                        </select>
+                    </td>
+                </tr>
+            {% else %}
+                <input type="hidden" name="calendar" value="{{ calendar.cid }}" />
+            {% endif %}
+            <tr>
+                <td class="trow2" style="vertical-align: top;"><strong>{{ lang.event_date }}</strong></td>
+                <td class="trow1">
+                    <script type="text/javascript">
```

Comment:
A note for @justinsoltesz @Eric-Jackson - would be nice to take this inline JS out. The less inline JS we have, the better in my opinion.

Reply:
Would be great if we could have none so that admins can use a CSP without `unsafe-inline`.

Reply:
Yeah same goes for inline CSS imo, which is happening after the conversion to Twig I believe. Not sure if it would be better to remove JS now, or wait until we're making the changes to the HTML for a responsive theme.